app/views/protocols/form/study_form_sections/_study_information.html.haml

```diff
@@ -27,6 +27,13 @@
       = t(:protocols)[:studies][:information][:subtext]
   .panel-body.container-fluid
     .form-group.row
+      = form.label :research_master_id, class: 'col-lg-2 control-label rm-id' do
+        = link_to t(:protocols)[:studies][:information][:research_master_id], RESEARCH_MASTER_LINK
+      .col-lg-10
+        = form.text_field :research_master_id, class: 'form-control'
+        = form.hidden_field :has_human_subject_info, value: false, class: 'has-human-subject-info'
```

Comment:
Have to add validation to a RMID on the fly. This is a hidden field set to default to false. If a user checks the Human Subjects Info checkbox, sets this field to true which then will require presence validation on the RMID attribute.